src/main/java/com/github/mc1arke/sonarqube/plugin/almclient/azuredevops/model/Comment.java

```diff
@@ -19,7 +19,7 @@ public Comment(@JsonProperty("content") String content, @JsonProperty("author")
                    @JsonProperty("commentType") CommentType commentType) {
         this.content = content;
         this.author = author;
-        this.commentType = commentType;
+        this.commentType = CommentType.TEXT;
```

Comment:
This doesn't seem right to me: we're ignoring the parameter passed into the constructor and hardcoding a value, which means we never use the value actually returned by the API. Additionally this class is only used on API responses - not requests - so shouldn't impact whether Azure DevOps sends emails or not.

I'd expect the CreateCommentRequest class to have a `commentType` field added.